base/applications/regedit/framewnd.c

```diff
@@ -1146,12 +1146,8 @@ FreeObjectPicker(IN IDsObjectPicker *pDsObjectPicker)
     pDsObjectPicker->lpVtbl->Release(pDsObjectPicker);
 }
 
-/*******************************************************************************
- *
- *  FUNCTION: _CmdWndProc(HWND, unsigned, WORD, LONG)
- *
- *  PURPOSE:  Processes WM_COMMAND messages for the main frame window.
- *
+/*
```

Comment:
```suggestion
/**
```
This is a Doxygen comment.

Reply:
Katayama review1